src/components/dashboard/main-content/course-enrollments/tests/enrollment-testutils.js

```diff
@@ -31,7 +32,8 @@ const createCourseEnrollmentWithStatus = (
 };
 
 const createRawCourseEnrollment = (options) => ({
-  courseRunId: 'course-v1:Best+course',
+  courseKey: 'edX+DemoX',
+  courseRunId: 'course-v1:Best+course+2T2025',
```

Comment:
nit: might be good to have the course key be more reflective of the course run identifier. E.g., the `courseRunId` technically contains the `courseKey` as a substring, but this mock data defines the course key as `edX+DemoX` when the `courseRunId` technically references a course with the `Best+course`.